src/ConnectedMode/UI/ManageBinding/ManageBindingViewModel.cs

```diff
@@ -199,4 +221,28 @@ internal bool LoadConnections()
         RaisePropertyChanged(nameof(ConnectionSelectionCaptionText));
         return succeeded;
     }
+    
+    private async Task DisplayBindStatusAsync()
+    {
+        var solutionName = await solutionInfoProvider.GetSolutionNameAsync();
+        var isFolderWorkspace = await solutionInfoProvider.IsFolderWorkspaceAsync();
+        SolutionInfo = new SolutionInfoModel(solutionName, isFolderWorkspace ? SolutionType.Folder : SolutionType.Solution);
+        
+        var boundServerProject = connectedModeServices.ConfigurationProvider.GetConfiguration()?.Project;
+        var connection = boundServerProject?.ServerConnection;
```

Comment:
Please rename the variable connection to "serverConnection", because we also have Connection type and ConnectionInfo type... And it is very easy to confuse them all.